### PR TITLE
eo2464-hw0

### PR DESCRIPTION
**Pull request by egeozguroglu**

### CODE REVIEW

**Review by catw101 — CHANGES_REQUESTED** [review]
Restore `tkp2108.md`

**Comment by egeozguroglu** [comment]
Hi, I restored tkp2108.md.

File: tkp2108.md [hunk 1]
```diff
@@ -4,7 +4,9 @@ Name: Tim Paine
 
 UNI: tkp2108
 
-Github: [link](https://github.com/timkpaine)
+GitHub: [link](https://github.com/timkpaine)
```
Comment (timkpaine) [inline tkp2108.md:7]: Unnecessary modifications in this PR

**Review by timkpaine — CHANGES_REQUESTED** [review]

**Comment by egeozguroglu** [comment]
Hi Tim, I restored tkp2108.md fully to remove the changes I mistakenly made. The only diff of this PR is my .md file currently.

**Pull request merged**